scripts/qesap/test/unit/test_qesap_lib_cmd_ansible.py

```diff
@@ -0,0 +1,37 @@
+from unittest import mock
+import logging
+import os
+
+
+from lib.cmds import export_ansible_output
+
+log = logging.getLogger(__name__)
+
+
+def test_export_ansible_output():
+    '''
+    This test checks if the ansible execution generates log files
+    in this case generates the generic one "ansible.testAll.log.txt"
+    scripts/qesap/test/unit/test_qesap_ansible.py
+    
+    '''
+
+    test_dir = os.getcwd()
+    test_file = os.path.join(test_dir, 'ansible.testAll.log.txt')
```

Comment:
What is the purpose of 
```
'test1':'test_value1', 'test2':'test_value2', 'test3':'test_value3', 'test4':'test_value4',
```
?

Is it to check that `test_export_ansible_output` ignore them?

Reply:
yes